lib/vega_lite/data.ex

```diff
@@ -12,10 +14,19 @@ defmodule VegaLite.Data do
   alias VegaLite, as: Vl
 
   @doc """
-  Returns the specification for a given data, a mark and a list of fields to be encoded.
+  Returns the specification for a given data and a list of fields to be encoded.
+  """
+  def chart(data, fields), do: chart(Vl.new(), data, fields)
+
+  @doc """
+  Returns the specification for the given arguments.
+
+  It can takes a data, a mark and a list of fields to be encoded or
+  a valid `VegaLite` specification, a data and a list of fields to be encoded.
 
-  Each argument is accepted as the argument itself or a tuple with the argument and a keyword
-  list of options. All options must follow the specifications of the `VegaLite` module.
+  Each argument that is not a `VegaLite` specification nor a data, is accepted as the argument
+  itself or a keyword list of options. All options must follow the specifications of the
+  `VegaLite` module.
```

Comment:
This pattern is common, so I wouldn't worry about it. Usually what we do in this case is though is to document only one of the cases. So I would document this as only passing the mark and have all examples of passing `Vl.new()` as argument in `chart/4`.